scripts/pr-statistics.sh

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env bash
+
+
+# PARAMETERS ----------------------------
+GITHUB_REPO="status-im/status-react"
+
+
+# PLATFORM ----------------------------
+case "$OSTYPE" in
+  darwin*)  OS="DARWIN" ;;
+  *)        OS="LINUX" ;;
+esac
+
+# FUNCTIONS ----------------------------
+github_pr() {
+  repo="$1"
+  list_epoch=""
+
+	last_page=$(curl -i -sL "https://api.github.com/repos/$repo/pulls?sort=created&direction=asc&state=open" | grep rel=\"last\" | cut -d "," -f 2 | cut -d "&" -f 4 | cut -d "=" -f 2 | cut -d ">" -f 1)
+  [ "$last_page" = "" ] && last_page=1
+  for i in $(seq 1 $last_page); do
+    json="$(curl -sL https://api.github.com/repos/$repo/pulls?sort=created&direction=asc&state=open&page=$i)"
+		list_dt_created="$list_dt_created $(echo "$json" | grep -B2 "closed_at" | grep "created_at" | tr -d ' ' | cut -d "\"" -f 4)"
+
+    for dt in $list_dt_created; do
+      [ "$OS" = "DARWIN" ] && list_epoch="$list_epoch $(date -j -u -f '%Y-%m-%dT%H:%M:%SZ' "$dt" +%s)" \
+        || list_epoch="$list_epoch $(date -d "$dt" +%s)"
+    done
+  done
+
+  echo "$list_epoch"
+}
+
+median_list() {
+  list="$@"
+  len_list="$#"
+  median=""
+
+  (( middle_list = len_list / 2 ))
+  (( middle_list_next = middle_list + 1 ))
+
+  j=1
+  for l in $list; do
+    [ $j -eq $middle_list ] && median=$l
+    [ $j -eq $middle_list_next ] && median=$(echo "scale=2; ($median + $l)/2" | bc -l)
+    (( j = j + 1 ))
+  done
+  echo "$median"
+}
+
+five_nb_summary() {
+  list="$@"
+  len_list="$#"
+  sample_minimum=""
+  lower_quartile=""
+  median=""
+  upper_quartile=""
+  sample_maximum=""
+
+  # NOTE list is already sorted asc
+  i=1
+  for l in $list; do
+    [ $i -eq 1 ] && sample_minimum=$l
+    [ $i -eq $len_list ] && sample_maximum=$l
+    (( i = i+1 ))
+  done
+  median="$(median_list $list)"
+
+  i=1
+  for l in $list; do
+    # Q1
+    [ "1" = "$(echo "$l < $median" | bc -l)" ] && q1_list="$q1_list $l"
+    # Q3
+    [ "1" = "$(echo "$l > $median" | bc -l)" ] && q3_list="$q3_list $l"
+  done
+
+  lower_quartile="$(median_list $q1_list | cut -d '.' -f 1)"
+  median="$(echo $median | cut -d '.' -f 1)"
+  upper_quartile="$(median_list $q3_list | cut -d '.' -f 1)"
+
+  if [ "$OS" = "DARWIN" ]; then
```

Comment:
Column format here too, can we also add days open? Something like

Mon Nov 20 22:28:45 UTC 2017 (45 days)